### PR TITLE
Improve english grammar

### PR DESCRIPTION
**Pull request by FranDelaCasa**

### CODE REVIEW

**Comment by carlosvillu** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # SUI-Autocompleted
 
-`sui-autocompleted` is a simple React component to show a list of suggest under a input file, when you start to write something.
+`sui-autocompleted` is a simple React component that shows a list of suggestions under an input file when you start to write something.
 
 
 ## Install
 
-To run from the source and start to play with the examples:
+In order of running the source code and start to play with the examples:
```
Comment (delacruz-dev) [inline README.md]: Shouldn't it be `in order to` instead of `in order of`?

**Comment by FranDelaCasa** [comment]
@scm-spain/react-devs, review, please ;)

File: README.md (hunk 1 above)
Comment (FranDelaCasa) [inline README.md]: I think you're right! in order to + verb
I'm changing it in 2 minutes

Comment (FranDelaCasa) [inline README.md]: "outdated diff" :)

**Comment by delacruz-dev** [comment]
:+1:

**Pull request merged**